config/src/config/consensus_config.rs

```diff
@@ -55,7 +56,7 @@ pub enum ConsensusProposerType {
     // Pre-specified proposers for each round,
     // or default proposer if round proposer not
     // specified
-    RoundProposer(HashMap<Round, AccountAddress>),
+    RoundProposer(RoundProposerConfig),
```

Comment:
We should probably add to the comment, this is only for testing and not for prod like you did for RoundProposer below.